### DISCUSSION TITLE
Microphone type

### DISCUSSION THREAD

**Discussion by mapm1998** [seerge/g-helper#724] [Ideas]
I don't know if this is possible, but in the AC app, in my case (2022 M16) I have 3 definitions for the micro, and it would be great if it was possible to incorporate those options on the G-Helper.

**Comment by seerge** [comment]
@mapm1998 hello, doing this within app is not realistic (https://github.com/seerge/g-helper/issues/18) as that part is proprietary software.

But you can just check what Realtek Control panel shows you, on may new models it lets you choose microphone mode
